### ISSUE TITLE
How to make the training dataset, or any GT image data set to share?

### ISSUE THREAD

**Issue by Jeffrey98-AI** [TobiasGruening/ARU-Net#2]
Great job! 
I just cannot figure out how to prepare the training data set ,is there any tools to make the GT image?

**Comment by af258963** [comment]
I also want to know how to make training data.

**Comment by TobiasGruening** [comment]
Hey,
 how the train data should look like is described in the readme.

One possibility to set up such GT data is decribed in http://arxiv.org/abs/1802.03345
(Alg. 1 on page 10).

Kind regards,
Tobias

**Comment by ghost** [comment]
@TobiasGruening I have read your paper.
- Other than Aletheia & Transkribus, What tools do you use to label the baseline regions?
- How do you convert the labeled layout files into pixel ground truth images?
